### PR TITLE
IngredientsList: initial commit.

### PR DESCRIPTION
**Pull request by mfocshaner**
Container for currently chosen ingredients.

### CODE REVIEW

File: app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class IngredientsList {
+
+    private HashMap<String, Integer> ingredientsMap;
+
+    public IngredientsList() {
```
Comment (AmitSilber) [inline app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java]: "this" is redundant

File: app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        this.ingredientsMap = new HashMap<>();
```
Comment (AmitSilber) [inline app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java]: "this" is redundant

File: app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package com.huji.foodtricks.foodtricks;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class IngredientsList {
```
Comment (AmitSilber) [inline app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java:7]: add documentation

**Review by AmitSilber — APPROVED** [review]

File: app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java (hunk 3 above)
Comment (mfocshaner) [reply app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java:7]: v

File: app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java (hunk 2 above)
Comment (mfocshaner) [reply app/src/main/java/com/huji/foodtricks/foodtricks/IngredientsList.java]: v

**Pull request merged**